nncf/api/compression.py

```diff
@@ -254,6 +254,13 @@ def strip_model(self, model: ModelType) -> ModelType:
         """
         return model
 
+    @property
+    def compression_rate(self) -> float:
```

Comment:
I believe that the compression_level and compression_rate are similar attributes. Can we leave one of them?

Reply:
Not really, addressed by a more appropriate naming in #721